BrowserStack.java

```diff
@@ -15,15 +14,17 @@
  *         isEmpty()    - return true if stack is not empty else false
  *
  */
+import java.util.*;
 public class BrowserStack {
 
-    String[] myStack;
+    ArrayList<String> myStack = null;
```

Comment:
Using ArrayList is ok, but, it was not required. String[] is sufficient for this case.